src/client/components/ProfileImage/ProfileImage.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import useFetch from '../../hooks/useFetch';
+
+const ProfileImage = (username) => {
```

Comment:
Aren't these supposed to be `props`? And then later you use `props.userName`?
Also, why not use `userId` instead of searching by username - user id is unique, two users might have same user names.

Reply:
I had created this to add the profile image when there was only a dummy component for messageList and messages were not being fetched from the backend. In the meantime the messageList component was modified and due to some confusion the profile image was also added by someone else.
So Benjamin just wanted to see my approach through this PR...he said since the other is PR is already merged I just have to leave this and label it as "do not Merge".